src/main/webapp/WEB-INF/views/aboutus.jsp

```diff
@@ -19,14 +19,8 @@
     <link rel="stylesheet" href="/css/main.css">
   </head>
   <body>
-    <nav>
-      <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-        <li><a href="/feed.html">Public Feed</a></li>
-        <li><a href="/stats.html">Stats</a></li>
-      </ul>
-    </nav>
+    <%@ include file="./components/navigation.jsp" %>
```

Comment:
woohoo! thanks so much for the nav bar. Really saving all of our sanity

Reply:
Thanks so much for doing this +1 !!!